controller/controller.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+
+function homePage()
+{
+    require('./view/homeView.php');
+}
```

Comment:
:warning: 
Some files don't have new line at the end. For several reasons we add a new line in all files.